Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_35.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Microsoft Graph Security
+
+- Updated the Docker image to: *demisto/crypto:1.0.0.4834757*.
+- Fix a bug where multiple serviceSource supplied, the query filter is broken. The fix will use the $in operator instead of $eq with AND
```

Comment:
```suggestion
- Fixed an issue where supplying multiple *serviceSource* values caused the query filter to break. The fix now uses the `$in` operator instead of combining `$eq` with `AND`.
```